packages/components/src/internal/components/base/utils.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from 'react';
+
+import { ACTIVE_JOB_INDICATOR_CLS } from '../pipeline/constants';
+
+import { Tip } from './Tip';
+
+export function getTitleDisplay(content: ReactNode, hasActivePipelineJob: boolean): ReactNode {
```

Comment:
nit: Rather than creating this new file `internal/components/base/utils.tsx` I would consider putting this functional component in `internal/components/pipeline/utils.tsx` as it seems to pertain mostly to pipeline considerations/interactions.